torchci/scripts/backfillJobs.mjs

```diff
@@ -123,29 +124,67 @@ const queuedJobs = await client.queries.query({
   sql: {
     query: `
 SELECT
-    j.id
+    j.id,
+    w.repository.name as repo_name,
+    w.repository.owner.login as owner
 FROM
     workflow_job j
     INNER JOIN workflow_run w on j.run_id = w.id
 WHERE
     j.status = 'queued'
     AND w.status != 'completed'
     AND PARSE_TIMESTAMP_ISO8601(j.started_at) < (CURRENT_TIMESTAMP() - INTERVAL 5 MINUTE)
-    AND w.repository.name = 'pytorch'
+    AND w.head_repository.owner.login = 'pytorch'
 ORDER BY
     j._event_time DESC
 LIMIT 10000
 `,
   },
 });
 
-ids = queuedJobs.results.map((r) => r.id);
-
 // See above for why we're awaiting in a loop.
-for (const id of ids) {
+for (const {id, repo_name, owner} of queuedJobs.results) {
   await backfillWorkflowJob(
     id,
+    repo_name,
+    owner,
     (job) => job.status === "queued" && job.steps.length === 0
   );
 }
 console.log("::endgroup::");
+
+console.log("::group::Backfill unclassified logs...");
+const unclassifiedJobs = await client.queries.query({
+  sql: {
+    query: `
+select
+    j.id,
+from
+    commons.workflow_job j
+    join commons.workflow_run w on w.id = j.run_id
+where
+    j.torchci_classification is null
+    and w.head_branch = 'master'
+    and j.conclusion in ('failure', 'cancelled')
+    and PARSE_TIMESTAMP_ISO8601(j.completed_at) > CURRENT_DATETIME() - INTERVAL 30 MINUTE
+    and j.name != 'ciflow_should_run'
+    and j.name != 'generate-test-matrix'
+    and w.event != 'workflow_run'
+    and w.event != 'repository_dispatch'
+    and w.head_repository.full_name = 'pytorch/pytorch'
+`,
+  },
+});
+console.log(`There are ${unclassifiedJobs.results.length} jobs with unclassified logs`)
+for (const job of unclassifiedJobs.results) {
+  console.log(`Attempting to backfill log of ${job.id}`);
+  try {
+    const a = await request(
+      `https://vwg52br27lx5oymv4ouejwf4re0akoeg.lambda-url.us-east-1.on.aws/?job_id=${job.id}`
```

Comment:
Could you double check to see if this domain works?  I'm seeing that it's also used by `github-status-test` lambda.  So I wan to see if this might just be an internal lambda-to-lambda URL

Reply:
yes, works locally and in ci